e2e/cypress.json

```diff
@@ -8,5 +8,6 @@
     "reporterOptions": {
         "mochaFile": "results/results-[hash].xml",
         "toConsole": false
-    }
+    },
+    "testFiles": "**/*_spec.js"
```

Comment:
This ensures that if we have helper code inside of our specs folders that only the specs will get run.